centrallix/expression/exp_functions.c

```diff
@@ -3098,8 +3069,7 @@ int exp_fn_max(pExpression tree, pParamObjects objlist, pExpression i0, pExpress
 	    tree->String[0] = '\0';
 	    tree->Integer = 0;
 	    tree->Types.Double = 0;
-	    tree->Types.Money.FractionPart = 0;
-	    tree->Types.Money.WholePart = 0;
+	    tree->Types.Money.Value = 0;
```

Comment:
Should max be tested?

Reply:
Yes, max() and min() should be tested, last summer the test suite guys focused on the string functions.